sync.config.js

```diff
@@ -1,11 +1,11 @@
-let mongoId = process.env.REACT_APP_ID;
+let mongoId = 'process.env.REACT_APP_ID';
```

Comment:
Changing this file is out of the scope of your PR so I'd just go ahead and revert to the version on main. To save without formatting, use Cmd + K  + S.